appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/impl/PayaraMicroImpl.java

```diff
@@ -732,6 +733,17 @@ public boolean isNoCluster() {
         return noCluster;
     }
 
+    /**
+     * Indicated whether distributed data grid is enabled
```

Comment:
```suggestion
     * Indicates whether Hazelcast is enabled
```

Reply:
As per FISH-372, the option --nohazelcast requires to disable only the distributed data grid, while keeping available any local one. Hence, the line #737 should read "Indicated whether distributed data grid is enabled"